package.json

```diff
@@ -28,23 +28,29 @@
       "url": "https://github.com/nikoskalogridis"
     }
   ],
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
+  "main": "./v3/index.js",
+  "module": "./v3/index.mjs",
+  "types": "./v3/index.d.ts",
```

Comment:
dev: point to `./v3` instead of `./dist` since we now have a dedicated build folder for the v3 files.